src/olympia/promoted/admin.py

```diff
@@ -0,0 +1,83 @@
+from django.contrib import admin
+from django.forms.models import modelformset_factory
+
+from olympia.addons.models import Addon
+from olympia.discovery.admin import SlugOrPkChoiceField
+
+from .forms import AdminBasePromotedApprovalFormSet
+from .models import PromotedAddon, PromotedApproval
+
+
+class PromotedApprovalInlineChecks(admin.checks.InlineModelAdminChecks):
+    def _check_relation(self, obj, parent_model):
+        """PromotedApproval doesn't have a direct FK to PromotedAddon (it's via
+        Addon, Version) so we have to bypass this check.
+        """
+        return []
+
+
+class PromotedApprovalInline(admin.TabularInline):
+    model = PromotedApproval
```

Comment:
@bobsilverberg this is for PromotedApproval

Reply:
I ran your patch locally, but couldn't see where I could maintain approvals. Where does it exist in the admin?

Reply:
Inlines are at the bottom of the page, but you can only view the approvals (and delete).  The approval will be done via the reviewer tools.  If you wanted to see this locally you'd do `PromotedApproval.objects.create(version=<version of the addon you've created the PromotedApproval for>, group_id=<a group of your choice, e.g. RECOMMENDED).id)` in a django shell.

Reply:
Great, thanks @eviljeff!